### ISSUE TITLE
Explore running limited migration database to answer ad-hoc queries.

### ISSUE THREAD

**Issue by adunkman** [ustaxcourt/ef-cms#772]
## Description

We'd like to the ability to run limited ad-hoc queries against the Blackstone Database -- In case we need to retrieve any information or just to make it more accessible. It's also more cost-effective than the EC2 instance dedicated to running the Microsoft SQL database. 

We're currently using the help of AWS Pro Services to migrate the information into a MySQL RDS Instance.

## Tasks
- [ ] Mike to Verify with Mike McVicker whether or not we really need this / how long it would take to stand back up from S3 since it's used so sporadically.
- [ ] Complete Migration to AWS MySQL RDS instance
- [ ] Gain access to AWS MySQL RDS instance
  - [ ] Need credentials 
  - [ ] Need to know what VPC / network in order to gain access
- [ ] Turn off Microsoft SQL Server EC2 instance

**Comment by mark-meyer** [comment]
Currently, legacy data is hosted in a different court environment in a SQL Server database running inside a large ec2 instance. This was provisioned in a way to allow migrations to run quickly, but is over-provisioned for occasional ad hoc queries. Also, the current database contains the binary data for the PDFs, which require a lot of storage and are probably no longer needed. One proposed solution is to move the legacy data to a smaller database hosted on RDS without the PDF binary columns, we could save money while still allowing for occasional queries as needed.

**Comment by mark-meyer** [comment]
Jan 19 meeting with AWS Services, Mike McVicker, and Mike Marcotte: AWS will explore options for loading a subset of the redacted migration SQL Server tables into a lightweight RDS instance, which will allow small ad-hoc queries of data going forward.

**Comment by mmarcotte** [comment]
Mark to figure out how to get access to that RDS MySQL database.

**Comment by mmarcotte** [comment]
Mark can connect, but no credentials yet. Coming soon!

**Comment by mmarcotte** [comment]
Mike Marcotte to get credentials for the database. See if we can close this ticket.

**Comment by mmarcotte** [comment]
Let's get clarity about what specifically has been migrated into MySQL.

**Comment by mmarcotte** [comment]
It appears it's everything without redacted documents. Mark cannot have access. 

The RDS instance is not publicly available, and secured via a VPC. I'm currently working with Mike McVicker to try and gain access.

**Comment by mmarcotte** [comment]
Moved to blocked as it appears the migration from Microsoft SQL tables is still in progress. Waiting on AWS Pro Services to complete the operation.

**Comment by adunkman** [comment]
There are a few questions about the need for this now that we can query DAWSON’s data using Athena — we may be able to solve this issue by closing it. 

One of the Mikes would be the decision maker.

**Comment by mmarcotte** [comment]
Information getting stored away into S3 to be resuscitated if need be. Abandoning RDS instance that would probably not be used very often if at all.
